src/lib/components/drs/formComponents/DrsObjectParents.js

```diff
@@ -26,6 +26,7 @@ const DrsObjectParents = props => {
                     </Typography>
                 </div>
             }
+            activeDrsObject={{id: props.id}}
```

Comment:
can you change this prop to `activeDrsObjectId` since it's not the full `activeDrsObject` being passed?